src/Config.ts

```diff
@@ -86,7 +86,10 @@ export default function ClientConfig(opts: StreamrClientOptions = {}) {
         // Ethereum and Data Union related options
         // For ethers.js provider params, see https://docs.ethers.io/ethers.js/v5-beta/api-providers.html#provider
         mainnet: undefined, // Default to ethers.js default provider settings
-        sidechain: 'https://rpc.xdaichain.com/',
+        sidechain: {
+            url: 'https://rpc.xdaichain.com/',
+            chainId: 100
+        },
```

Comment:
Does this need merging in `options: StrictStreamrClientOptions `?

If not maybe should add a comment explicitly saying that it shouldn't be merged so it's not added later accidentally. Otherwise probably should merge:
```typescript
        sidechain: {
            ...defaults.sidechain,
            ...opts.sidechain
        },
        ```

Reply:
It should not merge